### ISSUE TITLE
Hello, a question about the CDAN function in loss.py

### ISSUE THREAD

**Issue by BaeHann** [agrija9/Deep-Unsupervised-Domain-Adaptation#7]
Hello!
   Within the CDAN function, entropy.detach() was not found. Does that mean the entropy will be affected (as well as the classification result and relevant parameters in model) when loss.backward() was executed?

**Comment by BaeHann** [comment]
I am not that proficient in English. If there is anything grammatically wrong or incomprehensible, please be so kind as to inform me.
